prefect_docker/scripts/configuration/settings.toml

```diff
@@ -38,8 +38,9 @@ debug = false
 DATAVERSE_URL="https://portal.staging.odissei.nl"
 DATAVERSE_IMPORTER_URL="https://dataverse-importer.labs.dans.knaw.nl"
 PUBLICATION_DATA_UPDATER_URL="https://dataverse-date-updater.labs.dans.knaw.nl"
-METADATA_FETCHER_URL="https://dataverse-fetcher.labs.dans.knaw.nl"
+METADATA_FETCHER_URL="http://dataverse-metadata-fetcher:8083"
 DATAVERSE_MAPPER_URL="https://dataverse-mapper.labs.dans.knaw.nl"
 DANS_TRANSFORMER_SERVICE="https://transformer.labs.dans.knaw.nl"
 EMAIL_SANITIZER_URL="https://emailsanitizer.labs.dans.knaw.nl/sanitize"
+METADATA_REFINER_URL="http://metadata-refiner:7878"
```

Comment:
This url looks like it is for development.

Reply:
Fixed